publisher/publisher.go

```diff
@@ -0,0 +1,64 @@
+package publisher
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/screwdriver-cd/sd-cmd/config"
+	"github.com/screwdriver-cd/sd-cmd/screwdriver/api"
+	"github.com/screwdriver-cd/sd-cmd/util"
+)
+
+// Publisher is a type to publish sdapi and sdstore.
+// It receives strings which input by a user.
+// If -f option is valid, yaml file will be loaded to commandSpec struct.
+type Publisher struct {
+	specPath    string
+	commandSpec *util.CommandSpec
+}
+
+// Run is a method to publish sdapi and sdstore.
+func (p *Publisher) Run() error {
+	sdAPI := api.New(config.SDAPIURL, config.SDToken)
+	specPath := p.specPath
+	specResponse, err := sdAPI.PostCommand(specPath, p.commandSpec)
```

Comment:
I thinks you do not need to declare specPath. 
```
specResponse, err := sdAPI.PostCommand(p.specPath, p.commandSpec)
```

Reply:
Thank you! I've fixed.